src/db.c

```diff
@@ -1082,7 +1082,8 @@ void copyCommand(client *c) {
     robj *o;
     redisDb *src, *dst;
     int srcid;
-    long long dbid, expire;
+    int dbid;
```

Comment:
Should we put them with the same type in one line (and below) ?

Reply:
Thanks, I was already thinking about in one line.